test/e2e/playwright/transfer-orders.spec.ts

```diff
@@ -134,16 +155,16 @@ test('test', async ({ page }) => {
     .press('Escape');
   await page.getByTestId(`button-accordion-cancel-button-${firstOrderId}`).click();
   await expect(page.getByTestId(`validated-cases-row-0`)).not.toBeVisible();
-  await expect(page.getByTestId(`new-case-input-${firstOrderId}`)).toBeDisabled();
 
-  // TODO CAMS-269 skipped failing assertion. Look into suggested case id
-  // await expect(page.getByTestId(`new-case-input-${firstOrderId}`)).toHaveValue(
-  //   firstOrder.docketSuggestedCaseNumber,
-  // );
-  await expect(page.getByLabel(`Select new court`).locator('visible=true')).toHaveValue('');
+  await page.getByTestId('suggested-cases-radio-empty').click();
+  await expect(page.getByLabel(`New court`).locator('visible=true')).toHaveValue('');
+  await expect(page.getByTestId(`new-case-input-${firstOrderId}`)).toBeDisabled();
+});
 
-  const page1Promise = page.waitForEvent('popup');
-  await page.getByTestId(`approved-transfer-original-case-link-${firstOrder.caseId}-link`).click();
-  const page1 = await page1Promise;
-  expect(page1.url()).toContain(`/case-detail/${firstOrder.caseId}/`);
+test.skip('test opening link to case detail screen', async () => {
```

Comment:
I may be missing something here as to why opening the case detail screen doesn't work anymore.